packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Jumio/selectors.js

```diff
@@ -0,0 +1,21 @@
+import { selectors } from 'data'
+import { lift, path } from 'ramda'
+import { createDeepEqualSelector } from 'services/ReselectHelper'
+
+export const getData = createDeepEqualSelector(
```

Comment:
creating a deep equal selector is no real use here for two reasons:
1. Selects and operations are lightning fast it's not O(n^3) on a large list
2. Selection would rarely be done more then once